lm_optimizer.py

```diff
@@ -52,7 +52,8 @@ def objective(trial):
 
 
 def main(_):
-    initialize_globals()
+    Config = _SttConfig()
+    initialize_config_globals(Config)
```

Comment:
It looks like you can implement the `initialize_config_globals` in the `def _post_init_()` function of `_SttConfig` if you want to make things even shorter.